### DISCUSSION TITLE
Docke compose on Debian LXC: failed to register layer: unlinkat /home/ubuntu: invalid argument

### DISCUSSION THREAD

**Discussion by insalata-fresca** [photoprism/photoprism#3909] [Q&A]
Hi guys I am having torubles in running the docker compse on an lxc container
It gives always this error:
failed to register layer: unlinkat /home/ubuntu: invalid argument
![image](https://github.com/photoprism/photoprism/assets/113389438/2e8f4223-1897-4bc0-b77b-b2a8c7d2fd02)

any clue on wht this can be?

I also tried to install on a ubuntu container and have same error
I am following the official guide:
https://docs.photoprism.app/getting-started/docker-compose/

**Comment by lastzero** [comment]
From what I know, Docker is a fork of LXC. So they use the same technology (both are not VMs) and just like you wouldn't run Docker containers in Docker containers, you (probably) shall not run Docker in/with LXC:
- https://docs.photoprism.app/getting-started/faq/#lxc-images

**Reply by insalata-fresca** [reply]
Ia gree with what you say, but I still do not understand the above error and where it comes from

**Reply by lastzero** [reply]
I don't use LXC, but it makes perfect sense that you can't run Docker Compose (or Docker in general) within LXC. There may also be naming/namespace conflicts on the same host. Unfortunately, I can't go into detail as I'm an application developer and not a Linux kernel developer.

We provide these packages to run PhotoPrism within an LXC container:
- https://dl.photoprism.app/pkg/linux/README.html

For a complete (third-party) LXC install script / example, see:
- https://github.com/tteck/Proxmox/blob/main/install/photoprism-install.sh
